frontend/pages/media/blog/112.js

```diff
@@ -0,0 +1,174 @@
+import React, { Component } from "react";
+import PageHeader from "../../../components/Header";
+import { loadBlogPreviews } from "../../../lib/api";
+import styles from "../../../styles/BlogPost.module.css";
+import AuthorCard from "../../../components/AuthorCard";
+import BlogSuggestions from "../../../components/BlogSuggestions";
+import ShareBtns from "../../../components/ShareBtns";
+
+const blogPost112 = ({ blogPreviews }) => {
+  // start webpage at the top
+  // useEffect(() => {
+  //   window.scrollTo(0, 0);
+  // }, []);
+
+  return (
+    <div>
+      {/* Cover Photo */}
+      <PageHeader imgUrl="/headers/blog-header.jpg" title="Blog Post #112" />
+      {/*End of Header*/}
+
+      {/*Start of blog post*/}
+      <div className={styles.blogContent}>
+        <div className={styles.blogHeader}>
+          <p className={styles.date}>September 29, 2022</p>
+          <h2 className={styles.blogTitle}>
+            Historical Women in Tech: How the women of the past shaped our
```

Comment:
can you serve this file as a [dynamic page](https://nextjs.org/learn/basics/dynamic-routes) instead of using a manual page (see past events as reference)